sdk/ml/azure-ai-ml/azure/ai/ml/_schema/_deployment/batch/batch_job.py

```diff
@@ -58,6 +59,13 @@ def make(self, data: Any, **kwargs: Any) -> Any:
                         data[EndpointYamlFields.BATCH_JOB_INPUT_DATA][key] = UriFileJobInput(uri=input_data.path)
                     elif input_data.type == AssetTypes.URI_FOLDER:
                         data[EndpointYamlFields.BATCH_JOB_INPUT_DATA][key] = UriFolderJobInput(uri=input_data.path)
+
+        if data.get(EndpointYamlFields.BATCH_JOB_OUTPUT_DATA, None):
+            for key, output_data in data[EndpointYamlFields.BATCH_JOB_OUTPUT_DATA].items():
+                if isinstance(output_data, Output):
+                    if output_data.type == AssetTypes.URI_FILE:
```

Comment:
Why only uri file? What about uri folders?

Reply:
We don't have support to uri_folders in the current swagger version we have. Will create a following PR to include them